server/src/test/java/org/opensearch/extensions/rest/ExtensionRestRequestTests.java

```diff
@@ -129,6 +141,54 @@ public void testExtensionRestRequest() throws Exception {
         }
     }
 
+    public void testExtensionRestRequestWithNoIdentityToken() throws Exception {
```

Comment:
Nit: this test cases is full of fluff already verified in testExtensionRestRequest, I'd recommend trimming down to parts where the behavior is different / could be different

Reply:
Will do.